File: LeetCode_3759/Main.py
class Solution(object):
    def countElements(self, nums, k):
        """
        :type nums: List[int]
        :type k: int
        :rtype: int
        """
        n = len(nums)
        if not k:
            return n
        
        nums.sort()
        cnt = 0
        for num in nums:
            if num < nums[n - k]:
                cnt += 1
        return cnt

File: LeetCode_3759/Main.cpp
#include<bits/stdc++.h>
using namespace std;

class Solution {
public:
    int countElements(vector<int>& nums, int k) {
        sort(nums.begin(), nums.end());

        int n = nums.size();
        if (k == 0) return n;
        int cnt = 0;
        for (int num : nums) {
            if (num < nums[n - k]) {
                cnt += 1;
            }
        }
        return cnt;
    }
};